net/dnsviz/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+# $Id$
+
+PortSystem         1.0
+
+PortGroup          github 1.0
+PortGroup          python 1.0
+
+github.setup       dnsviz dnsviz 0.6.5 v
+github.tarball_from releases
+
+categories         net
+platforms          darwin
+license            GPL-2+
+supported_archs    noarch
+maintainers        deccio.net:casey
+
+description        DNS measurement and diagnostic tool suite
+long_description \
+  DNSViz is a tool suite for measurement, diagnostic, and visualization of \
+  Domain Name System (DNS) behavior, including its security extensions \
+  (DNSSEC).
+
+checksums          rmd160 16a8b7094670e8c0012bbe362f52f6fb9828a3f9 \
+                   sha256 4598476625b2cd81224bc908d6fb98086c9568460d2074e35320fd651dadb269
+
+depends_lib        port:py27-dnspython \
```

Comment:
Are these just runtime dependencies or are they required during the build? Given that the port is `noarch`, I suspect that runtime (or potentionally build) dependency would be sufficient.

You would usually use:
```tcl
depends_run-append port:py${python.version}-dnspython
```
`append` is there to avoid removing dependency on Python that was added by the python PortGroup.